client/me/purchases/manage-purchase/index.tsx

```diff
@@ -1823,10 +1823,10 @@ function getCancelPurchaseNavText(
 
 	if ( isDomainRegistration( purchase ) ) {
 		text = translate( 'Cancel domain' );
-	} else if ( isPlan( purchase ) ) {
-		text = translate( 'Cancel plan' );
 	} else if ( isSubscription( purchase ) ) {
 		text = translate( 'Cancel subscription' );
+	} else if ( isPlan( purchase ) ) {
+		text = translate( 'Cancel plan' );
```

Comment:
I don't think I understand when we'd have a plan that isn't also a subscription... Should this just be removed instead?